ui/src/containers/Resources/index.tsx

```diff
@@ -17,17 +18,20 @@ import './Resources.css';
 const Resources = () => {
   const { resources } = useMst();
 
-  const checkResources = (resources: IResource[]) => {
-    return !resources.length ? (
+  const checkResources = (resourceList: IResource[]) => {
+    return !resourceList.length ? (
```

Comment:
Could you please consider writing a unit test for different conditions ??

Reply:
I guess for that there is a test already

Reply:
I think there is test for empty state, I guess we need to write for `Clear All Filters` as well